proto/provenance/metadata/v1/query.proto

```diff
@@ -21,96 +21,198 @@ service Query {
     option (google.api.http).get = "/provenance/metadata/v1/params";
   }
 
-  // Scope returns a specific scope by id
+  // ---- Primary Data Queries -----
+
+  // Scope searches for a scope.
+  //
+  // The scope id, if provided, must either be scope uuid, e.g. 91978ba2-5f35-459a-86a7-feca1b0512e0 or a scope address,
+  // e.g. scope1qzge0zaztu65tx5x5llv5xc9ztsqxlkwel. The session addr, if provided, must be a bech32 session address,
+  // e.g. session1qxge0zaztu65tx5x5llv5xc9zts9sqlch3sxwn44j50jzgt8rshvqyfrjcr. The record_addr, if provided, must be a
+  // bech32 record address, e.g. record1q2ge0zaztu65tx5x5llv5xc9ztsw42dq2jdvmdazuwzcaddhh8gmu3mcze3. Case doesn't
+  // matter.
```

Comment:
Case is defined by the specifications of bech32 encoding to be all lower case, so in that sense it does matter even if we don't require it to match thei requirement.